worker/src/test/java/gov/cms/ab2d/worker/processor/JobProcessorIntegrationTest.java

```diff
@@ -254,6 +254,7 @@ void when_errorCount_is_below_threshold_do_not_fail_job() {
         assertEquals(100, event.getBenesExpected());
         assertEquals(CONTRACT_NAME, event.getContractNumber());
         assertEquals(100, event.getBenesSearched());
+        assertEquals(100, event.getBenesWithEobs());
```

Comment:
Reusing the value of 100 open up a cross-wire vulnerabilty i.e. if one has two fields a and b and sets them both to the same value, then one can not detect if b was updated when a should be.  

Consider using unique values for each member field.